src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/config/JurisdictionToUserMappingTest.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.config;
+
+import org.assertj.core.api.Assertions;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Import;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.idam.Credential;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
+@Import(JurisdictionToUserMapping.class)
+@EnableConfigurationProperties
+@TestPropertySource(properties = {
+    "idam.users.sscs.username=bulkscanorchestrator+systemupdate@gmail.com",
+    "idam.users.sscs.password=Password12"
+})
+public class JurisdictionToUserMappingTest {
+
+    @Autowired
+    JurisdictionToUserMapping mapping;
+
+    @Test
+    public void should_parse_up_the_properties_into_map() {
+        Credential creds = mapping.getUser("SSCS");
+        assertThat(creds.getPassword()).isEqualTo("Password12");
+        assertThat(creds.getUsername()).isEqualTo("bulkscanorchestrator+systemupdate@gmail.com");
+    }
+
+    @Test
+    public void should_return_empty_optional_if_not_found() {
+        Throwable throwable = Assertions.catchThrowable(() -> mapping.getUser("NONE"));
+        assertThat(throwable).isInstanceOf(UserNotConfiguredException.class);
+        assertThat(throwable).hasMessage("No configured user for jurisdiction:NONE found");
```

Comment:
This would be much nicer on junit 5
Could upgrade in a separate PR?
https://github.com/hmcts/idam-java-client/blob/master/build.gradle#L165
https://github.com/hmcts/idam-java-client/blob/master/build.gradle#L150
https://github.com/hmcts/idam-java-client/blob/master/src/test/java/uk/gov/hmcts/reform/idam/client/IdamClientTest.java#L46

Reply:
Test method name doesn't seem to match what's being tested